src/components/BuddiesContext.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useEffect, useState } from "react";
+import { Buddy } from "./WorkBuddies/BuddyInterface";
+
+interface BuddiesContextType {
+  contextData: Buddy[] | undefined;
+  setData: (data: Buddy[]) => void;
+}
+
+export const BuddiesContext = createContext<BuddiesContextType>({
+  contextData: [],
+  setData: (data: Buddy[]) => {},
```

Comment:
This can be simplified like so:
 ```suggestion
  setData: () => {},
```